apps/odyssey-stats/src/widget/mini-chart.jsx

```diff
@@ -66,23 +68,47 @@ const MiniChart = ( { siteId, quantity = 7, gmtOffset, odysseyStatsBaseUrl } ) =
 		return () => observer.disconnect();
 	} );
 
+	const isEmptyChart = ! chartData.some( ( bar ) => bar.value > 0 );
+	const placeholderChartData = Array.from( { length: 7 }, () => ( {
+		value: Math.random(),
+	} ) );
+
 	return (
 		<div
 			ref={ chartWrapperRef }
 			id="stats-widget-minichart"
 			className="stats-widget-minichart"
 			aria-hidden="true"
 		>
-			<div className="stats-widget-minichart__chart-head">
-				<Intervals selected={ period } compact={ false } onChange={ setPeriod } />
-			</div>
 			{ isLoading && <StatsModulePlaceholder className="is-chart" isLoading={ true } /> }
 			{ ! isLoading && (
 				<>
-					<Chart barClick={ barClick } data={ chartData } minBarWidth={ 35 }>
+					{ ! isEmptyChart && (
+						<div className="stats-widget-minichart__chart-head">
+							<Intervals selected={ period } compact={ false } onChange={ setPeriod } />
+						</div>
+					) }
```

Comment:
What happens when there's no chart data for the last seven days, but they have stats data available for the last seven weeks? Would hiding the interval selector make sense in such a context?

Reply:
I had a look at the API response - there's no difference between "when a site is new and doesn't have stats" and "it's an old site and not data in the past few days". So I think we should think of a new message, as the wording only makes sense when user firstly started to use Jetpack Stats. Would it make sense to state:

`No data found in the period. If your site is new, this chart will show you details about your views and visitors once stats become available. You could [Learn more about stats](https://jetpack.com/stats/) or if you think this is an error, [contact our awesome support team](https://jetpackme.wordpress.com/contact-support/).`

@jsnmoon @ederrengifo @dognose24

Reply:
@jsnmoon Yeah, we should keep the `Intervals` for the period changing even though the current data is empty.

@kangzj Agreed! The message could be more comprehensive for unpopular old sites. cc @ederrengifo

Reply:
> Once stats become available, this chart will show you details about your views and visitors. Learn more about stats

I'm okay with our current copy, as it's ambiguous enough to apply to new and old sites. Stats can "become available" for both contexts.

Reply:
@jsnmoon  I find `Once stats become available` not very accurate for old sites as Stats is available, just no data 🤔

Reply:
I personally interpret it to mean "stats will become available once there's anything to show" in my head 🙂

Reply:
cc @ederrengifo and @keoshi for your thoughts on the copy!

Reply:
We can tackle the copy as a follow-up PR later if there are other determined thoughts!

Reply:
We've gone ahead with the current copy and merged the PR. We can always come back and revise the copy if you folks have other thoughts. Thanks. 😄 

@ederrengifo @keoshi